sensor_msgs/include/sensor_msgs/impl/point_cloud2_iterator.hpp

```diff
@@ -149,8 +149,8 @@ inline void PointCloud2Modifier::resize(size_t width, size_t height)
   size_t size = width * height;
   resize(size);
 
-  cloud_msg_.width = width;
-  cloud_msg_.height = height;
+  cloud_msg_.width = static_cast<uint32_t>(width);
```

Comment:
Isn't it better to change `PointCloud2Modifier::resize(size_t width, size_t height)` to receive uint32_t?

Reply:
Maybe. I'm not sure if that would require explicit cases for `size` on L150.

Reply:
@tonylitianyu Can you update the signature of the new API to use type `uint32_t` for arguments?